etcdctl/ctlv2/command/util.go

```diff
@@ -156,7 +156,12 @@ func getTransport(c *cli.Context) (*http.Transport, error) {
 		CertFile: certfile,
 		KeyFile:  keyfile,
 	}
-	return transport.NewTransport(tls, defaultDialTimeout)
+
+	to := c.GlobalDuration("total-timeout")
```

Comment:
why shall we do this? say if the total-timeout is 10 seconds and the defaultDialTimeout is 5 seconds, the change might cause the dailtimeout to be rewrote by a longer one which is 10 seconds. this is incorrect.